control-plane/agents/src/bin/ha/cluster/switchover.rs

```diff
@@ -195,9 +195,66 @@ impl SwitchOverRequest {
             {
                 error!(volume.uuid=%self.volume_id, %error, "Cancelling switchover");
                 self.stage = Stage::Errored;
-                Err(error)
+                Err(anyhow!(error.to_string()))
```

Comment:
Why is this change needed?

Reply:
In the same arm i had errors returned using anyhow! formatting if unable to get nvme_path and uri formatting. I was getting error that match arm had incompatible  return types.